test/adapter/json_api/relationships_test.rb

```diff
@@ -103,23 +103,29 @@ def test_relationship_simple_link
                 self: '//example.com/link_author/relationships/bio'
               }
             }
-            assert_relationship(:bio, expected)
+
+            author = @author.dup
```

Comment:
interesting way to solve the problem.

Does minitest run tests in `//` ? Why isn't `@author` re-set before each of these tests (in setup)? (my lack of minitest knowledge is showing)